vcfmerge.c

```diff
@@ -2564,6 +2564,9 @@ void gvcf_write_block(args_t *args, int start, int end)
     {
         int slen  = 0;
         char *seq = faidx_fetch_seq(args->gvcf_fai,maux->chr,out->pos,out->pos,&slen);
+        if (!seq)
+            exit(1); // faidx_fetch_seq has already reported the error.
```

Comment:
Better to use `error()` to explain what happened.

Reply:
It doesn't need it as faidx_fetch_seq has already used `hts_log_error` to explain the problem.  See the text in #1900:

```
[E::faidx_adjust_position] The sequence "chr2" was not found
Segmentation fault
```

It's the second line this is fixing. :-)